### PR TITLE
Exclude unnecessary files from packages

### PR DESCRIPTION
**Pull request by paescuj**
## Description

I noticed that there are quite a few files in the published packages which aren't necessary.
For example test files, but also whole `src` folders or files like `.editorconfig`.
This pull request updates the [files](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files) field in the `package.json` files to exclude those.

Note: Files like `README.md` and `LICENSE` are always included, therefore they don't have to be specified explicitly.

Also there are some packages with source map files. I wonder whether we really want these files in the published package or if it is sufficient if they are available in the local development environment?

## Type of Change

- [ ] Bugfix
- [x] Improvement
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [x] All tests are passing locally
- [x] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated. PR:

### CODE REVIEW

File: packages/specs/package.json [hunk 1]
```diff
@@ -36,8 +36,6 @@
 	"homepage": "https://github.com/directus/directus#readme",
 	"files": [
 		"dist",
-		"LICENSE",
-		"README.md",
```
Comment (rijkvanzanten) [inline packages/specs/package.json:40]: Isn't the readme required for rendering on npm, or does it send the readme no matter what?

Comment (paescuj) [reply packages/specs/package.json:40]: Yes, the readme ([as well as other files](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#files)) is always included 👍

**Review by rijkvanzanten — APPROVED** [review]

**Comment by paescuj** [comment]
@rijkvanzanten I've noticed that there are also some packages with source map files included. Do you think we need those files in the published packages or would it be sufficient if they are available in the local development environment only?

**Comment by rijkvanzanten** [comment]
@paescuj Lets get rid of them 👍🏻

**Comment by paescuj** [comment]
> @paescuj Lets get rid of them 👍🏻

Yup, makes sense IMO 👍

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**